src/api.js

```diff
@@ -15,3 +15,10 @@ export const getArticle = (articleId) => {
     return res.data.article;
   });
 };
+
+export const getCommentsPerArticle = (articleId) => {
+  return newsApi.get(`/articles/${articleId}/comments`).then((res) => {
+    console.log(res.data)
```

Comment:
Don't forget to remove console.log before merging into main (avoids them clogging up the console when other run your project)